src/components/Checkbox/Checkbox.stories.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Checkbox } from "./Checkbox";
+import { withKnobs } from "@storybook/addon-knobs";
+
+export const Primary = () => <Checkbox></Checkbox>;
```

Comment:
`Primary` may not have that much sense here - maybe we could leave `Playground` here only and give it some knobs? for example to change the `value`? wdyt?